packages/components/src/collection/src/useScrollableCollection.ts

```diff
@@ -64,6 +66,20 @@ export function useScrollableCollection(containerRef: RefObject<Element>, nodes:
                     height += outerHeight;
                 });
 
+                if (boundaryElement) {
+                    const { offsetHeight: boundaryHeight } = boundaryElement;
+
+                    const { y: containerY } = containerRef.current.getBoundingClientRect();
+
+                    // If the container and direction is bottom is overflowing the boundary, adjust the height
```

Comment:
This sentence doesn't make sense to me.. are you trying to say this:
```suggestion
                    // If the direction is `bottom` and the container is overflowing the boundary, adjust the height.
```